### PR TITLE
Database

### PR DESCRIPTION
**Pull request by fadnesscharlie**
Added PSQL Database

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "lint": "eslint .",
     "test": "jest --coverage",
-    "dev": "NODE_ENV=development node index.js"
+    "dev": "NODE_ENV=development node index.js",
+    "route": "node ./src/routes/apiRoutes.js"
```
Comment (MuckT) [inline package.json]: We should test this from jest or via running the server locally.

File: package.json [hunk 2]
```diff
@@ -22,8 +23,12 @@
   },
   "homepage": "https://github.com/follow-the-nft/follow-the-nft-api#readme",
   "dependencies": {
+    "axios": "^0.24.0",
     "dotenv": "^10.0.0",
-    "express": "^4.17.1"
+    "express": "^4.17.1",
+    "sequelize": "^6.12.0-beta.1",
+    "sequelize-cli": "^6.3.0",
```
Comment (MuckT) [inline package.json]: Remove as `sequelize-cli` should not be a dev dependency but installed locally as a global.
`npm uninstall sequelize-cli && npm install sequelize-cli -g`

File: prepWork/databaseModel.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+
+# Database Model Schema
+
+Below is our Database Model. We will be storing data in our model from our API.
```
Comment (MuckT) [inline prepWork/databaseModel.md:4]: Please remove as we won't be storing nft data in our database - will fetch from chain.

File: src/models/collection-class.js [hunk 4]
```diff
@@ -0,0 +1,65 @@
+"use strict";
+
+class Collection {
```
Comment (MuckT) [inline src/models/collection-class.js]: We can remove the collection class as we won't have that we will only have the user model and user routes which require db updates.

File: src/models/index.js [hunk 5]
```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+require('dotenv').config();
+const { Sequelize, DataTypes } = require('sequelize');
+// const userModel;
+const nft = require('./nft-model.js');
```
Comment (MuckT) [inline src/models/index.js]: Can remove nft and collection models. We should build the user model in here.

File: src/routes/apiRoutes.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+const findByCollection = async (asset_owner) => {
+  // let getCollectionData = await axios.get(`https://api.opensea.io/api/v1/collections?asset_owner=${asset_owner}&offset=0&limit=20`)
+  let getCollectionData = await axios.get(`https://api.opensea.io/api/v1/collections?asset_owner=102384&offset=0&limit=300`)
+  console.log(getCollectionData.data)
+  getOneAsset(getCollectionData)
+}
+
+// console.log(axios.get(`https://api.opensea.io/api/v1/assets?order_direction=desc&offset=0&limit=3`).then(data => console.log(data.data.assets[0].collection)))
+
+findByCollection('102384')
```
Comment (MuckT) [inline src/routes/apiRoutes.js]: Lets remove the findByCollection hard coded call.

**Review by MuckT — CHANGES_REQUESTED** [review]

File: prepWork/databaseModel.md (hunk 3 above)
Comment (fadnesscharlie) [reply prepWork/databaseModel.md:4]: We need to have a database diagram, lets talk more about what we would use in place of this

File: package.json [hunk 7]
```diff
@@ -27,7 +27,6 @@
     "dotenv": "^10.0.0",
     "express": "^4.17.1",
     "sequelize": "^6.12.0-beta.1",
-    "sequelize-cli": "^6.3.0",
     "sqlite3": "^5.0.2"
```
Comment (MuckT) [inline package.json]: This should be a dev dependency.

**Review by MuckT — APPROVED** [review]
LGTM!

**Review by jeremywbrazell — APPROVED** [review]

**Pull request merged**